### DISCUSSION TITLE
Job Announcement: New Orleans Jazz & Heritage Foundation

### DISCUSSION THREAD

**Discussion by relyons** [digipres/digipres.github.io#43] [General]
New Orleans Jazz & Heritage Foundation Archive

Helis Foundation Digital Collections Manager 
The New Orleans Jazz & Heritage Archive is seeking a full-time on-site Digital Collections Manager (DCM) to lead the continued implementation, maintenance and expansion of the Archive’s digital preservation and access program which was started in 2019. The DCM is responsible for the acquisition, description, storage, preservation, and access of all digital materials in accordance with the established policies and procedures of the archive and the archival profession.  These responsibilities will be carried out by working with the New Orleans Jazz & Heritage Foundation and its partners to facilitate transfer and care of digital materials to the Archive’s network storage for preservation and access.  Oversight and maintenance of the hybrid on-prem/cloud storage system, as well as file preservation and integrity, lies with the DCM who works with archival staff to provide timely and efficient access to the archives’ digital content.

The DCM reports to the head Archivist and as a vital player in the Archive works with other staff to create and implement operational procedures; assists researchers and volunteers; and assists with physical materials. The DCM will contribute to the writing of grants, articles, and other activities that promote the accessibility of the Archive’s collections, the Archive itself, and Louisiana music, culture and heritage.  The DCM also works to develop and further the holdings of the Archive by working to identify and acquire relevant digital materials for it.

Minimum Qualifications

- Bachelor or Master’s degree in Library Science, Information Science, Computer Science or related field with a minimum of 3-5 years of experience managing digital collections.
- Knowledge of metadata standards and digital preservation best practices.
- Experience with administration of data storage systems, including network and systems administration, backup and recovery procedures, and network security protocols.
- Excellent communication and collaboration skills, with the ability to work effectively with a wide range of stakeholders with various skill levels.
- Ability to manage multiple projects and priorities simultaneously and to work independently as well as part of a team.
- Possession of excellent organizational, analytical, and technological skills.

Preferred Qualifications

- Experience with managing network attached storage devices such as those designed by Synology and QNAP.
- Expertise with database technologies such as MySQL or PostgresSQL.
- Fluency with cloud service providers and digital asset management systems, such as Backblaze, Wasabi, Digital Ocean, Discovery Garden Islandora, and Starchive 
- Knowledge of Louisiana music and culture.

Deadline to Apply is Thursday, August 31, 2023
Salary: $55,000 - $69,000
Those interested should apply by submitting a resume and cover letter to: [hr@jazzandheritage.org](mailto:hr@jazzandheritage.org)
For more information and see a complete job description, salary, and benefits go to:  https://www.jazzandheritage.org/about-us/career-opportunities/

**Comment by anjackson** [comment]
Hi, sorry, this forum is intended specifically for discussing matters related to the digipres.org shared resources. Please use the other established forums for job postings etc.
